model_hub/Makefile

```diff
@@ -77,23 +77,15 @@ publish-transformers-dev:
 
 .PHONY: build-mmdetection-dev
 build-mmdetection-dev:
-	if [[ "$(DEFAULT_GPU_IMAGE)" != *"cuda-11.3"* ]]; then \
-	  echo "Expected $(DEFAULT_GPU_IMAGE) to have cuda version $(MMCV_CUDA_VERSION)."; \
-	  exit 1; \
-	elif [[ "$(DEFAULT_GPU_IMAGE)" != *"pytorch-1.12"* ]]; then \
-	  echo "Expected $(DEFAULT_GPU_IMAGE) to have pytorch version $(TORCH_VERSION)."; \
-	  exit 1; \
-	else \
-	  docker build -f docker/Dockerfile.mmdetection \
-	  	  --build-arg BASE_IMAGE=$(DEFAULT_GPU_IMAGE) \
-	  	  --build-arg MMCV_CUDA_VERSION=$(MMCV_CUDA_VERSION) \
-	  	  --build-arg TORCH_VERSION=$(TORCH_VERSION) \
-	  	  --build-arg MMCV_VERSION=$(MMCV_VERSION) \
-	  	  --build-arg MMDETECTION_VERSION=$(MMDETECTION_VERSION) \
-	  	  --build-arg MODEL_HUB_VERSION=$(VERSION) \
-	  	  -t $(MMDETECTION_ENVIRONMENT_ROOT):$(SHORT_GIT_HASH) \
-	  	  .; \
-	fi;
+	docker build -f docker/Dockerfile.mmdetection \
```

Comment:
I see three different classes of thing here, could you please explain how they're related? It'd help me understand this PR.

1. MD-219
2. bumpenvs for environment images
3. the additional code changes

Reply:
Environments work almost always results in 2 PRs. One for environments changes, the second for bumpenvs. I tag the ticket to the bumpenvs (because I hate Jira automation.)

1. MD-219 is the environments feature that this PR is concerned with
2. Bumpenvs is the procedure that changes our images to target images that were published by that environments branch.
3. Additional code changes are a mix of dealing with the new naming conventions (model hub) and moving to new images (see the deepspeed skips)